uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -116,16 +113,16 @@ define([
        * reset dates, day types, change balance.
        */
       vm.changeInNoOfDays = function () {
-        reset();
+        this._reset();
         //reinitialize opening balance
-        initAbsenceType();
+        initAbsenceType.call(this);
```

Comment:
yes, the init is called from child controller and therefore need the context to be continued to be passed